.github/labeler.yml

```diff
@@ -94,6 +94,10 @@ orchestrator:
   - src/pybind/mgr/test_orchestrator/**
   - qa/tasks/mgr/test_orchestrator_cli.py
 
+rook:
+  - doc/mgr/rook.rst
+  - src/pybind/mgr/rook/**
```

Comment:
* qa/suites/orch/rook
* qa/tasks/rook.yp
* do we have a workload script as well?

(please verify the paths)

Reply:
> * do we have a workload script as well?
What do you mean by workload script?

Reply:
guess it's something under `qa/workunits/workunits`.